test_common/harness/imageHelpers.h

```diff
@@ -255,10 +291,22 @@ void read_image_pixel(void *imageData, image_descriptor *imageInfo, int x,
 
     // Advance to the right spot
     char *ptr = (char *)imageData;
-    size_t pixelSize = get_pixel_size(format);
-
-    ptr += z * slice_pitch_lod + y * row_pitch_lod + x * pixelSize;
-
+    switch (format->image_channel_data_type)
+    {
+        case CL_UNSIGNED_INT_RAW10_EXT: {
+            ptr += z * slice_pitch_lod + y * row_pitch_lod + (x / 4) * 5;
```

Comment:
Can we use `RAW10_EXT_CLUMP_SIZE` and `RAW10_EXT_CLUMP_NUM_PIXELS` throughout instead? (Same for raw12)

Reply:
sure, I'll make the change